modules/spring-config/src/main/java/org/mule/runtime/config/internal/context/lazy/LazyMuleArtifactContext.java

```diff
@@ -175,7 +175,6 @@ public LazyMuleArtifactContext(MuleContext muleContext, ArtifactAst artifactAst,
     this.artifactProperties = artifactProperties;
     this.runtimeLockFactory = runtimeLockFactory;
 
-    initialize();
```

Comment:
What has to be done before the initialize is the ast validation. But according to the comment below the graph should be done after the initialize. Do we have a way to guaranteee that we validate the ast without macroexpansion?